www/docs/server/error-handling.md

```diff
@@ -43,6 +43,7 @@ tRPC defines a list of error codes that each represent a different type of error
 | PAYLOAD_TOO_LARGE     | Request entity is larger than limits defined by server.                                                                 | 413       |
 | METHOD_NOT_SUPPORTED  | The server knows the request method, but the target resource doesn't support this method.                               | 405       |
 | UNPROCESSABLE_CONTENT | The server understands the request method, and the request entity is correct, but the server was unable to process it.  | 422       |
+| I_AM_A_TEAPOT | The server refuses to brew coffee because it is, permanently, a teapot.  | 422       |
```

Comment:
I think that was forgotten

```suggestion
| I_AM_A_TEAPOT | The server refuses to brew coffee because it is, permanently, a teapot.  | 418       |
```

Reply:
ah damn yeah